uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/b2c/GoogleLoginComponentHandler.java

```diff
@@ -85,4 +89,20 @@ public void handleNextButton() {
             throw new AssertionError(e);
         }
     }
+
+    public void handleRecoveryEmail() {
+        if(this.RECOVERY_EMAIL_PROMPT_EXPECTED) {
```

Comment:
don't need `this` when referencing `static` fields

```suggestion
        if(RECOVERY_EMAIL_PROMPT_EXPECTED) {
```

Reply:
7f8bb0f9a0fe2fefa07f49e21972b82e77e2887f